fairseq/data/data_utils.py

```diff
@@ -187,8 +203,24 @@ def is_batch_full(num_tokens):
 
         batch.append(idx)
 
-    # if len(batch) > 0:
-    #     yield batch
+    if len(batch) > 0:
+        yield batch
+
+
+def batch_by_size_tpu(
```

Comment:
Make sure we follow the style of native code.
Not saying this isn't ... just be sure we are not doing "our own" style.

Reply:
yep, I'm familiar w/ FB formatting and I manually checked, it would be better if we had a formatter. I asked around but nobody gave me anything.